paper/report.aux

```diff
@@ -1,28 +0,0 @@
-\relax 
-\bibstyle{siam}
-\citation{repovs2011}
-\citation{repovs2012}
-\citation{repovs2011}
-\citation{repovs2012}
-\citation{repovs2012}
```

Comment:
These changes might cause some problems. But we don't have to worry about this now and we will fix these inconsistencies later.